superset-frontend/packages/superset-ui-core/src/query/types/Operator.ts

```diff
@@ -33,6 +33,7 @@ const BINARY_OPERATORS = [
   'NOT LIKE',
   'REGEX',
   'TEMPORAL_RANGE',
+  'TEXT_SEARCH',
```

Comment:
Let's just use `ILIKE` for consistency with how search works across the app, see my first comment.